module/src/main/scala/org/scalatestplus/play/ChromeFactory.scala

```diff
@@ -49,5 +55,22 @@ trait ChromeFactory extends BrowserFactory {
 /**
  * Companion object to trait `ChromeFactory` that mixes in the trait.
  */
-object ChromeFactory extends ChromeFactory
+object ChromeFactory extends ChromeFactory {
 
+  /**
+   * Creates a new instance of `ChromeOptions`.
+   *
+   * @param headless whether or not to run Chrome in headless mode
+   * @return a new instance of `ChromeOptions`
+   */
+  def createChromeOptions(headless: Boolean): ChromeOptions = {
```

Comment:
Maybe define this as:

```scala
def createChromeOptions(headless: Boolean, args: String*): ChromeOptions = {
  ...
}
```
So that users can customize the arguments like `--disable-gpu`.

Reply:
On second thought, I'm not sure if `createChromeOptions` should be extended in that manner.

Users are already able to customize the arguments as follows:

```scala
val options = createChromeOptions(headless = true)
options.addArguments("--some-extra-option")
```

In addition, `ChromeOptions` has some other customization methods like `addExtensions`, `setExperimentalOption`, `setPageLoadStrategy`, etc. However, IMHO,  there seem no strong reason for making an exception for `args`.

Maybe I should hide `createChromeOptions` as package private (i.e. add `package[play]`) because of the lack of generality required for public API. I'd like to get your thoughts on it.

Reply:
`ChromeOptions` class of the latest selenium has [`setHeadless(headless)`](http://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/chrome/ChromeOptions.html#setHeadless-boolean-) method and `.createChromeOptions(headless)` can (and probably should) be replaced with it after https://github.com/playframework/scalatestplus-play/pull/110/commits/39aaf891cb138dcd10925a4cb4c8fb06539b6c4d is merged.